pkg/schema/schema.json

```diff
@@ -109,6 +100,14 @@
                     ],
                     "description": "type defines the kind of detection you want to use. For example IP will find IP addresses, whereas Keywords will find predefined keywords."
                 },
+                "domains": {
+                    "description": "The list of domains and their subdomains which should be obfuscated in the output.",
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "minItems": 1
```

Comment:
hmm, it didn't generate that check- I'll debug it.

Reply:
ah it's in the wrong level, you need to pull it up one so it's side-by-side with the items

Reply:
I think the generated validation is still broken because it expects the field to always be of length 1, even when not specified.

Reply:
not sure it's properly defined in the RFC http://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.6.4.2

I'd leave it out then, if we desire the domains to be optional.